app/controllers/AdminUsers/employees_controller.rb

```diff
@@ -25,6 +25,30 @@ def destroy
       redirect_to admin_users_employees_path, notice: 'Employee was successfully destroyed.'
     end
 
+    def edit_number_of_available_kudos_for_all
+      render :edit_number_of_available_kudos_for_all, locals: { employees: Employee.all }
+    end
+
+    def update_number_of_available_kudos_for_all
```

Comment:
since this is custom action I would consider naming it something like:
```suggestion
    def add_available_kudos_for_all
```

Reply:
Changed :)